src/config/config.cpp

```diff
@@ -101,13 +103,29 @@ static NodeInfoConfig ParseNodeInfoConfig(const aos::common::utils::CaseInsensit
     return nodeInfoConfig;
 }
 
-static MigrationConfig ParseMigrationConfig(const aos::common::utils::CaseInsensitiveObjectWrapper& migration)
+static MigrationConfig ParseMigrationConfig(
+    const aos::common::utils::CaseInsensitiveObjectWrapper& migration, const std::vector<ModuleConfig>& moduleConfigs)
 {
     MigrationConfig config {};
 
     config.mMigrationPath       = migration.GetValue<std::string>("migrationPath");
     config.mMergedMigrationPath = migration.GetValue<std::string>("mergedMigrationPath");
 
+    for (const auto& moduleConfig : moduleConfigs) {
+        aos::common::utils::CaseInsensitiveObjectWrapper object(
+            moduleConfig.mParams.extract<Poco::JSON::Object::Ptr>());
```

Comment:
There's a constructor that accepts poco dynamic var that does extract  in the implementation:
`CaseInsensitiveObjectWrapper::CaseInsensitiveObjectWrapper(const Poco::Dynamic::Var& var)`

```suggestion
            );
```

Reply:
good. thank you. changed